files/templates/additional-scrape-configs.template.yaml

```diff
@@ -64,12 +69,15 @@
     cert_file: /etc/prometheus/secrets/{{ .SecretName }}/crt
     key_file: /etc/prometheus/secrets/{{ .SecretName }}/key
     insecure_skip_verify: false
+{{ end }}
   relabel_configs:
+{{ if not .IsInCluster }}
   - target_label: __address__
     replacement: {{ .APIServerURL }}:443
   - source_labels: [__meta_kubernetes_node_name]
```

Comment:
Is there any reason why we are not relabelling the metrics path even when we are inside the cluster?

Reply:
We might actually need this or at least the port to get the targets correctly scraped.

Reply:
Just tried this but we need to get the certificates for api server for this to work
<img width="1257" alt="image" src="https://user-images.githubusercontent.com/6536819/91172274-2bc8e080-e6dc-11ea-83f3-d96bb7d2170a.png">

Reply:
Do we not have them somewhere?